requirements.txt

```diff
@@ -26,4 +26,4 @@ pathlib2==2.3.5
 flask-marshmallow==0.10.1
 marshmallow-sqlalchemy==0.17.0
 boto3==1.10.39
-marshmallow==2.20.2
+marshmallow==2.20.2
```

Comment:
You shouldn't have any changes in this file, can you revert this?

Reply:
yeap